README.md

```diff
@@ -33,3 +33,6 @@ gulp.task('default', function () {
 });
 ```
 
+## Documenation
```

Comment:
You're missing a `t` in documentation here and in the paragraph below. Looks good besides that!

Reply:
d'oh! good catch